### PR TITLE
Add BlockSettle to compatibility matrix

### PR DESCRIPTION
**Pull request by BlockSettle**

### CODE REVIEW

File: _data/compatibility/blocksettle.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      caption: >
+        The RBF dialog allows the user to replace/change/remove the output(s)
+        to create a new transaction
+segwit:
+  tested:
+    date: "2020-10-29"
+    platforms:
+      - Windows
+      - macOS
+      - Linux
```
Comment (bitschmidty) [inline _data/compatibility/blocksettle.yaml:65]: Same suggest as above to Linux only.

File: _data/compatibility/blocksettle.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    send:
+      signals_bip125: "true"
+      list: "true"
+      details: "false"
+      shows_replaced_version: "true"
+      shows_original_version: "false"
+  examples:
+    - image: /img/compatibility/blocksettle/rbf/RBF_desktop_notification.png
+      caption: >
+        Desktop notification includes RBF signalling
```
Comment (bitschmidty) [inline _data/compatibility/blocksettle.yaml:29]: Suggestion to replace signalling with signaling as we standardized on that in the rest of the matrix.

File: _data/compatibility/blocksettle.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      notification: "true"
+      list: "true"
+      details: "true"
+      shows_replaced_version: "true"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "true"
+      list: "true"
+      details: "false"
+      shows_replaced_version: "true"
```
Comment (bitschmidty) [inline _data/compatibility/blocksettle.yaml:24]: Is there a screenshot that shows this distinction between shows_replaced_version vs shows_original_version ?

**Review by bitschmidty — CHANGES_REQUESTED** [review]
Thanks so much for putting the effort into this PR for the compatibility matrix. Ive provided some suggestions before merging.

For all of the images, if you could run `optipng` on each to keep image sizes small. One batch way to do that is:

`optipng -o7 img/compatibility/blocksettle/*/*`

**Comment by sergey-chernikov** [comment]
> Thanks so much for putting the effort into this PR for the compatibility matrix. Ive provided some suggestions before merging.
> 
> For all of the images, if you could run `optipng` on each to keep image sizes small. One batch way to do that is:
> 
> `optipng -o7 img/compatibility/blocksettle/*/*`

Thanks for your review! I'll reply to other comments shortly. As for this one, `optipng -o7` was already invoked for each image file committed, another invocation gives "image is already optimized". Could you please advise?

File: _data/compatibility/blocksettle.yaml (hunk 3 above)
Comment (sergey-chernikov) [reply _data/compatibility/blocksettle.yaml:24]: We have only a replaced TX shown, "shows_original_version" is set to "false" in both cases.

**Comment by bitschmidty** [comment]
Thanks for making those changes. I have marked the comments as resolved.

In terms of the image optimization, when I run the command, I get several images unoptimized. I have added a commit with the optimized images. Let me know if you have any questions.

The only other suggestion I have is that the RBF_right_click.png image seems to be a picture of the screen instead of a screenshot like the others. It would be nice to have it match your other images. Let me know if that is possible.

**Comment by sergey-chernikov** [comment]
Hi Mike!
Thank you very much for your contribution. As for RBF_right_click.png it's really a photo of the screen, because when trying to take a screenshot of a pop-up menu programmatically, that menu immediately disappears.

**Comment by jnewbery** [comment]
Thanks for this PR @sergey-chernikov. It looks ready to merge. Could you squash all commits into a single commit for merge please?

**Comment by sergey-chernikov** [comment]
Done

**Review by bitschmidty — APPROVED** [review]

**Comment by jnewbery** [comment]
This is still failing our linter. blocksettle.png is not optimized.

**Comment by bitschmidty** [comment]
Ive optimized the remaining logo image file, squashed.

**Comment by bitschmidty** [comment]
Merged!

@BlockSettle @sergey-chernikov Thanks for contributing!

**Comment by goatpig** [comment]
Thank you for the help. Much appreciated!

**Pull request merged**